core/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -230,7 +230,7 @@ private InterceptCallHandler(
     }
   }
 
-  private static <OReqT, ORespT, WReqT, WRespT> ServerMethodDefinition<WReqT, WRespT> wrapMethod(
+  static <OReqT, ORespT, WReqT, WRespT> ServerMethodDefinition<WReqT, WRespT> wrapMethod(
```

Comment:
And OReqT and ORespT here look unnecessary.

Reply:
This one looks necessary, because `wrapHandler` expects the first two argument's types to be bound, and the `?` from `getServerCallHandler` and the `?` from `getMethodDescriptor` would be treated as different even if they came from the same `ServerMethodDefinition<?,?>`.